en/docs/deploy/deploy-wso2-using/aws.md

```diff
@@ -1,8 +1,4 @@
----
```

Comment:
shall we change the folder name "deploy-wso2-using" to something like "deployment-options"